### ISSUE TITLE
STM32G0 misses EXTI because of HAL incompatibility.

### ISSUE THREAD

**Issue by suikan4github** [suikan4github/murasaki#117]
**Describe the bug**
A [HAL incompatibility](https://github.com/suikan4github/stm32-defects#d003-stm32g0-hal_gpio_exti_callback-incompatibility) prevents to catch the EXTI of Murasaki. 

**Expected behavior**
This problem is fixed by Issue #113. Need to track the problem.

**Comment by suikan4github** [comment]
The fix is installed [here](https://github.com/suikan4github/murasaki/blob/11736577772fbc308f2bc8fff086eebda4890b6e/Src/murasaki_callback.cpp#L463). 

```
/*
 * Following callback is to fix the incompatibility of the STM32G0 EXTI implementation.
 * See  https://github.com/suikan4github/murasaki/issues/123
 */

void HAL_GPIO_EXTI_Rising_Callback(uint16_t GPIO_Pin) {
    HAL_GPIO_EXTI_Callback(GPIO_Pin);
}

/*
 * Following callback is to fix the incompatibility of the STM32G0 EXTI implementation.
 * See  https://github.com/suikan4github/murasaki/issues/123
 */
void HAL_GPIO_EXTI_Falling_Callback(uint16_t GPIO_Pin) {
    HAL_GPIO_EXTI_Callback(GPIO_Pin);

}
```

**Comment by suikan4github** [comment]
This is already merged and published as v3.0.0
